docs/tutorial/development-environment.md

```diff
@@ -0,0 +1,114 @@
+# Developer Environment
+
+Electron development is essentially Node.js development. To turn your operating
+system into an environment capable of building desktop apps with Electron,
+you will merely need Node.js, npm, a code editor of your choice, and a
+rudimentary understanding of your operating system's command line client.
+
+## Setting up macOS
+
+> Electron supports Mac OS X 10.9 (and all versions named macOS) and up. Apple does
+not allow running macOS in virtual machines unless the host computer is already
```

Comment:
> virtual machines

This blockquote seems like it may not be applicable to the majority of folks who are reading the "Setting up macOS" docs. Seems like more of a concern when it's time to package the app, right?

Reply:
I've added this because three things kept coming up in my workshops and classes:

1) People coming in with ancient machines (think Windows XP and 10.8 Macs)
2) I could usually get those people up and running without replacing their whole machine by giving them a VM
3) That approach doesn't work on Macs 😢

I'm not married to it though, we can totally take it out.

Reply:
Fair enough! Leave it in. 👍